src/Analyzers/CSharp/CodeFixes/NamespaceSync/CSharpNamespaceSyncCodeFixProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CodeFixes.NamespaceSync;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.NamespaceSync
```

Comment:
i still don't love hte name namespacesync. it's veyr vague.  Is this: UpdateNamespaceNameToMatchFolderStructure?  Or UpdateFolderStrutureToMatchNamespaceName.